dipy/tracking/utils.py

```diff
@@ -108,6 +108,29 @@ def density_map(streamlines, affine, vol_dims):
     return counts
 
 
+def get_filtered_streamlines(streamlines, *, len=1):
+    """Filter streamlines based on their length.
+
+    Parameters
+    ----------
+    streamlines : sequence
+        A sequence of streamlines, where each streamline is represented as
+        an array of points.
+    length_threshold : int, optional
+        The minimum number of points a streamline must have to be included
+        in the result. Default is 1.
+
+    Returns
+    -------
+    filtered_streamlines : list
```

Comment:
Should be ArraySequence (Streamlines Object).

Reply:
if it is an `ArraySequence`, do not forget to take care of `data_per_point` and `data_per_streamline`.